File: cfgs/java_cfgs_foo/MINIMUM_CHARACTERS_ADDED_FRONT_MAKE_STRING_PALINDROME.java
Nodes
1;special;;;
2;statement;int l = fun_1();14;
3;statement;int i = 0;15;
4;statement;int j = foo_1 - 1;15;
5;conditional;foo_1 <= foo_2;16;
6;conditional;fun_1(foo_2) != fun_2(foo_4);18;
7;statement;return false;19;
8;exit;;;
9;statement;i++;17;
10;statement;j--;17;
11;statement;return true;22;
Edges
0;2;
1;3;
2;4;
3;5;
4;6;11;
5;7;9;
6;8;
7;
8;10;
9;5;
10;8;

File: cfgs/pythoncfgs/MINIMUM_CHARACTERS_ADDED_FRONT_MAKE_STRING_PALINDROME.py
Nodes
1;statement;foo_1 = fun_1(foo_2);8;
2;statement;foo_1 = 0;9;
3;statement;foo_1 = foo_2 - 1;10;
4;conditional;foo_1 <= foo_2;11;
5;statement;return true;16;
6;conditional;foo_1[foo_2] != foo_3[foo_4];12;
7;statement;foo_1 += 1;14;
8;statement;foo_1 -= 1;15;
9;statement;return false;13;
10;exit;;
Edges
0;2;
1;3;
2;4;
3;5;6;
4;10;
5;7;9;
6;8;
7;4;
8;10;
9;
